repositories/migrations/20250210144400_create_sanctions_whitelist.sql

```diff
@@ -0,0 +1,45 @@
+-- +goose Up
+-- +goose StatementBegin
+
+alter table sanction_check_configs
+    add column whitelist_field_ast
```

Comment:
We may want to harmonize this depending on how we end up calling those object IDs (see previous comment).